dkc/core/models/folder.py

```diff
@@ -82,6 +84,28 @@ def ancestors(self) -> models.query.RawQuerySet[Folder]:
             [self.pk, Folder.MAX_TREE_HEIGHT + 1],
         )
 
+    @transaction.atomic
+    def increment_size(self, amount: int) -> None:
+        """
+        Increment or decrement the Folder size.
+
+        This function increments (or decrements, if ``amount`` is negative) the Folder's current
+        size.
+
+        If this increment would exceed the max allotment for this quota, a ``ValidationError``
+        is raised and the transaction is rolled back.
```

Comment:
Was this comment copypasted from the quota branch? This last sentence should be removed, right?

Also this comment should mention that this increments the folder and all its ancestors.